server/sonar-web/src/main/js/api/ce.ts

```diff
@@ -20,6 +20,31 @@
 import { getJSON, post, RequestData } from '../helpers/request';
 import throwGlobalError from '../app/utils/throwGlobalError';
 
+export interface PendingTask {
+  componentId: string;
```

Comment:
I suggest to pretend that we don't have `componentId` field 😄 It should be deprecated & removed soon

Reply:
Ah yeah good idea!